package.json

```diff
@@ -92,7 +92,7 @@
     "npm": false
   },
   "volta": {
-    "node": "12.10.0",
+    "node": "12.22.12",
```

Comment:
One of the refreshed packages had an `engines` entry that wanted a newer Node version